cmd/subnetcmd/deploy.go

```diff
@@ -360,14 +422,23 @@ func useAllKeys(network models.Network) ([]string, error) {
 	return existing, nil
 }
 
-func loadCreationKey(network models.Network, keyName string) ([]string, error) {
-	path := filepath.Join(app.GetKeyDir(), keyName+constants.KeySuffix)
-	k, err := key.LoadSoft(network.NetworkID(), path)
+func loadCreationKey(network models.Network, kc keychain.Accessor) ([]string, error) {
+	addrs := kc.Addresses().List()
+	if len(addrs) == 0 {
+		return nil, fmt.Errorf("not creation addresses found")
+	}
+	addr := addrs[0]
+	networkID, err := network.NetworkID()
+	if err != nil {
+		return nil, err
+	}
+	hrp := key.GetHRP(networkID)
+	addrStr, err := address.Format("P", hrp, addr[:])
 	if err != nil {
 		return nil, err
 	}
 
-	return k.P(), nil
+	return []string{addrStr}, nil
```

Comment:
If we are always just returning one address, maybe this func could just return a `string` instead of a slice?